Jenkinsfile

```diff
@@ -167,9 +170,6 @@ pipeline {
         booleanParam(name: 'CI_UNIT_TEST',
                      defaultValue: true,
                      description: 'Run the Unit CI tests')
-        booleanParam(name: 'CI_FI_el8_TEST',
-                     defaultValue: true,
-                     description: 'Run the Fault Injection on EL 8 CI tests')
```

Comment:
Why do we want to remove this?

Reply:
Somehow this PR ended up with two entries for this parameter, so I removed one of them.